client/src/App.jsx

```diff
@@ -22,7 +22,16 @@ class App extends React.Component {
       sessionId: null,
       sessionName: '',
       sessionComplete: false,
-      splitMechanism: '',
+      splitMechanism: 'by Item',
+      tipPercentage: 0,
+      finalTotals: {
+        paymentsOwed: {},
+        preliminaryTotal: 0.00,
+        tipAmount: 0.00,
+        tax: 0.00,
+        finalTotal: 0.00
+      },
+      host: { name: 'host', order: [] },
```

Comment:
I think we are wanting to remove this

Reply:
@jabrake wanted it removed on an older component

Reply:
remove what? host?